core/src/scss/components/card/_card--minimal.scss

```
@@ -0,0 +1,22 @@
+.su-card--minimal {
+  @include box-shadow($depth: 'flat', $stroke: 'none');
+  background-color: inherit;
+
+  .su-card__contents {
+    // override modular spacing at all breakpoints
```

```suggestion
    // Override modular spacing at all breakpoints
```

Please to forgive my pedantry.